src/index.html

```diff
@@ -9,6 +9,40 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo_img">
+        <img src="\images\logo.png" alt="logo image">
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a href="#" class="nav_link is-active">Apple</a>
+          </li>
+          <li class="nav_item">
+            <a href="#" class="nav_link">Samsung</a>
+          </li>
+          <li class="nav_item">
+            <a href="#" class="nav_link">Smartphones</a>
+          </li>
+          <li class="nav_item">
+            <a href="#"
+               class="nav_link"
+               data-qa="hover">Laptops & Computers</a>
```

Comment:
```suggestion
            <a
              href="#"
             class="nav_link"
             data-qa="hover"
           >
             Laptops & Computers
           </a>
```

If you have three or more attributes in one tag then each of them should be in new line